templates/dob_jobapplications.yml

```diff
@@ -119,81 +119,81 @@ dataset:
       - nta
       - gisbin
     sql: |
-      SELECT 
-        jobnumber,
-        jobdocnumber,
-        jobtype,
-        jobdescription,
-        existingoccupancy,
-        proposedoccupancy,
-        existingnumstories,
-        proposednumstories,
-        existingzoningsqft,
-        proposedzoningsqft,
-        existingdwellingunits,
-        proposeddwellingunits,
-        jobstatusdesc,
-        latestactiondate,
-        prefilingdate,
-        assigned,
-        paid,
-        fullypaid,
-        approved,
-        fullypermitted,
-        signoffdate,
-        zoningdist1,
-        zoningdist2,
-        zoningdist3,
-        specialdistrict1,
-        specialdistrict2,
-        landmarked,
-        cityowned,
-        ownertype,
-        nonprofit,
-        ownerfirstname,
-        ownerlastname,
-        ownerbusinessname,
-        ownerhousestreetname,
-        zip,
-        ownerphone,
-        existingheight,
-        proposedheight,
-        totalconstructionfloorarea,
-        horizontalenlrgmt,
-        verticalenlrgmt,
-        enlargementsqfootage,
-        initialcost,
-        loftboard,
-        littlee,
-        curbcut,
-        cluster,
-        housenumber,
-        streetname,
-        bin,
-        borough,
-        block,
-        lot,
-        specialactionstatus,
-        latitude,
-        longitude,
-        existingzoningsqft,
-        proposedzoningsqft,
-        buildingclass,
-        otherdesc
-      FROM @filename
-      WHERE jobdocnumber = '01'
-      AND ( 
-        jobtype LIKE '%A1%' OR
-        jobtype LIKE '%DM%' OR
-        jobtype LIKE '%NB%'
-      )
+        SELECT 
+          ROW_NUMBER() OVER(PARTITION BY jobnumber ORDER BY SUBSTR(dobrundate, 7, 4)||SUBSTR(dobrundate, 1, 2)||SUBSTR(dobrundate, 4, 2) DESC) as gid,
```

Comment:
Looked at data on my machine, record with `gid=1` always has the later date. Once we figure out this issue with `_status_q.sql` then I think it's ready to merge